packages/docs/src/stories/components/combobox.stories.ts

```diff
@@ -0,0 +1,659 @@
+        </sd-combobox>
+      </div>
+      <script type="module">
+        // the highlight option renderer utility function can be imported via:
+        // import { highlightOptionRenderer } from '@solid-design-system/components';
+
+        // preview-ignore:start
+        const highlightOptionRenderer = ${optionRenderer};
+        // preview-ignore:end
+
+        const comboboxes = document.querySelectorAll('.highlight-combobox');
+        comboboxes.forEach(combobox => {
+          combobox.getOption = highlightOptionRenderer;
+        });
+      </script>
+    `;
+  }
+};
+
+/**
+ * Use to group `s visually.
+ */
+export const GroupingQuery = {
+  render: () => html`
+    <div class="h-[260px] w-[400px]">
+      <sd-combobox label="Group elements" value="g">
+        <sd-optgroup label="B">
+          ${createColorOptionHtml('Black')} ${createColorOptionHtml('Blue')} ${createColorOptionHtml('Brown')}
+        </sd-optgroup>
+        <sd-optgroup label="G"> ${createColorOptionHtml('Green')} ${createColorOptionHtml('Grey')} </sd-optgroup>
+        <sd-optgroup label="L"> ${createColorOptionHtml('Light Green')} </sd-optgroup>
+        <sd-optgroup label="M"> ${createColorOptionHtml('Magenta')} </sd-optgroup>
+        <sd-optgroup label="O"> ${createColorOptionHtml('Orange')} </sd-optgroup>
+        <sd-optgroup label="W"> ${createColorOptionHtml('White')} </sd-optgroup>
+        <sd-optgroup label="P"> ${createColorOptionHtml('Pink')} ${createColorOptionHtml('Purple')} </sd-optgroup>
+        <sd-optgroup label="R"> ${createColorOptionHtml('Red')} </sd-optgroup>
+        <sd-optgroup label="W"> ${createColorOptionHtml('White')} </sd-optgroup>
+        <sd-optgroup label="Y"> ${createColorOptionHtml('Yellow')} </sd-optgroup>
+      </sd-combobox>
+    </div>
+  `
+};
+
+/**
+ * The height of the filtered options list can be customized by setting the max-height on the listbox part of the combobox.
+ */
+export const SuggestionContainerHeight = {
+  render: () => html`
+    <div class="h-[260px] w-[400px]">
+      <style>
+        #max-height::part(listbox) {
+          max-height: 100px;
+        }
+      </style>
+      <sd-combobox label="Preferred Color" id="max-height" value="g"> ${createColorOptionsHtml()} </sd-combobox>
+    </div>
+  `
+};
+
+/**
+ * Use the `clearable` attribute to add a clear button that removes the selected value.
+ */
+
+export const Clearable = {
+  render: () => html`
+    <div class="w-[400px] h-[500px]">
+      <sd-combobox size="lg" label="Label" placement="bottom" clearable value="option-1">
+        <sd-option value="option-1">Option 1</sd-option>
+        <sd-option value="option-2">Option 2</sd-option>
+        <sd-option value="option-3">Option 3</sd-option>
+        <sd-option value="option-4">Option 4</sd-option>
+        <sd-option value="option-5">Option 5</sd-option>
+      </sd-combobox>
+    </div>
+  `
+};
+
+/**
+ * Use the `multiple` attribute to allow multiple options to be selected.
+ */
```

Comment:
The component should stay open when selecting an element:

See e. g. https://solid-design-system.fe.union-investment.de/docs/?path=/docs/components-sd-select--docs#multiple and https://vaadin.com/docs/latest//components/multi-select-combo-box